Buildings/Controls/OBC/FDE/PackagedRTUs/TSupSet.mo

```diff
@@ -0,0 +1,322 @@
+        Text(
+          extent={{-98,-20},{-42,-40}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid,
+          textString="When setback cooling mode
+is active the supply air temperature
+set point is set to the highest
+terminal unit space temperature
+minus a constant 10 degrees.",
+          horizontalAlignment=TextAlignment.Left),
+        Rectangle(
+          extent={{-366,-142},{-166,-210}},
+          lineColor={179,151,128},
+          lineThickness=0.5,
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-96,-122},{-40,-142}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid,
+          horizontalAlignment=TextAlignment.Left,
+          textString="When setback heating mode
+is active the supply air temperature
+set point is set to the fixed
+heating set point.")}),
+    Documentation(revisions="<html>
+<ul>
+<li>June 11, 2020, by Henry Nickels:<br>Reversed logic to logical switches.</li>
+<li>May 29, 2020, by Henry Nickels:<br>Internalize min, max, and heat setpoints as parameters.</li>
+<li>May 28, 2020, by Henry Nickels:<br>Removed CDN and WUP inputs as they were
+redundant to SBC and SBH. Changed cooling reset set point to percentage of total terminal units. </li>
+<li>May 22, 2020, by Henry Nickels:<br>First implementation. </li>
+</ul>
+</html>",
+      info="<html>
+<p>Calculation of the supply air temperature set point performed 
+by the BAS and transmitted to the factory controller. </p>
+<h4>Cooling Requests Reset</h4>
+<p>During normal occupied mode operation, the supply air temperature set point is reset between minimum 
+(<code>minSATsp</code>) and maximum 
+(<code>maxSATsp</code>) values based on terminal unit cooling requests. </p>
+<p>The total number of terminal unit cooling requests 
+(<code>totCoolReqs</code>) is a network input that totalizes all requests from terminal units. 
+The cooling request set point is calculated as a percentage of the total number of terminal units 
+(<code>TotalTU</code>) served by the RTU (e.g. 15% of 80 terminal units = 12 cool request set point). 
+The set point and requests are evaluated by a PI loop which outputs a value that increases from 0-1 as cooling requests increase.</p>
+<p>The PI loop output is input to a linear converter that outputs the reset set point 
+(<code>ySATsetpoint</code>).</p>
+<h4>Setback Cooling</h4>
+<p>During setback cooling mode (<code>SBC</code>) the supply air temperature set point is calculated from the highest space temperature 
+(<code>highSpaceT</code>) less a fixed offset of ten degrees. 
+(e.g. highest reported space temperature of (25 degC) - 10 = (15 degC) supply temperature set point). 
+The fixed value offset should be based on the unit cooling capacity per hour. </p>
```

Comment:
I don’t quite understand what the person who uses or configures this sequence needs to do based on 
> The fixed value offset should be based on the unit cooling capacity per hour.”

Do you mean the offset should be set to dT = Q/m/cp? This should be clearer explained.